apisix/control/v1.lua

```diff
@@ -269,6 +269,13 @@ local function iter_add_get_routes_info(values, route_id)
         if new_route.value.upstream and new_route.value.upstream.parent then
             new_route.value.upstream.parent = nil
         end
+        -- remove healthcheck info
+        new_route.checker = nil
```

Comment:
@nic-chen pls take a look when you have time

this changing seem fine to me, but I can not sure 100%

Reply:
No problem, I discussed it with @Revolyssup  yesterday